media/static-bundles.json

```diff
@@ -824,6 +824,13 @@
       ],
       "name": "firefox_welcome_page5"
     },
+    {
+      "files": [
+        "js/base/uitour-lib.js",
+        "js/firefox/privacy/messaging-experiment.js"
+      ],
+      "name": "firefox_welcome_page8"
```

Comment:
These pages are probably temporary but just in case they live for a while, let's rename this bundle to make it distinct from the welcome pages (since we're likely to have a welcome8 in the near future).